docs/user/README.md

```diff
@@ -89,11 +80,9 @@ To contact the Glean team you can:
 
 - Find us in the [#glean channel on chat.mozilla.org](https://chat.mozilla.org/#/room/#glean:mozilla.org).
 - Send an email to *glean-team@mozilla.com*.
-- The Glean SDK team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
+- The Glean team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
```

Comment:
I think we could safely remove Georg from this list.

Please note that I believe this is the "Glean SDKs" team still. The Glean team also include others (wlach and relud for example)

Reply:
I agree, thanks for catching this. 

I'll remove Georg too.